msvc2017/Parsers/Parsers.vcxproj

```diff
@@ -261,6 +262,10 @@
     <ClCompile Include="..\..\parse\UniverseObjectTypeValueRefParser.cpp" />
     <ClCompile Include="..\..\parse\ValueRefParser.cpp" />
     <ClCompile Include="..\..\parse\VisibilityValueRefParser.cpp" />
+    <ClCompile Include="..\..\parse\PythonParser.cpp" />
```

Comment:
this duplicates line 225, which breaks the project file

Reply:
after fixing that, seems to start ok with a702bfb7418a

Reply:
Fixed it